main.tex

```diff
@@ -12,6 +12,7 @@
 % TODO: change thesis information
 \newcommand*{\getUniversity}{Technische Universität München}
 \newcommand*{\getFaculty}{Informatics}
+\newcommand*{\getDegree}{Degree Course}
```

Comment:
```suggestion
\newcommand*{\getDegree}{Informatics}
```

Could you please change it to Informatics here since most of the students using this template are Informatics students.

Reply:
sure